src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -2714,14 +2714,13 @@ def ptq(
             verbose=get_param(calib_params, "verbose"),
         )
         q_util.register_skip_quantization(layer_names=get_param(selective_quantizer_params, "skip_modules"))
-        q_util.quantize_module(model)
 
         model = ptq(
             model,
             selective_quantizer=q_util,
-            calib_loader=calib_loader,
+            calibration_loader=calib_loader,
```

Comment:
Did we rename this parameter and properly deprecated it ?

Reply:
It isn't rename.
The former PR was merged with an error